whois-query/src/main/java/net/ripe/db/whois/query/acl/HazelcastPersonalObjectAccounting.java

```diff
@@ -1,55 +1,33 @@
 package net.ripe.db.whois.query.acl;
 
-import com.hazelcast.core.Hazelcast;
 import com.hazelcast.core.HazelcastInstance;
-import com.hazelcast.core.IMap;
 import com.hazelcast.core.OperationTimeoutException;
+import com.hazelcast.map.IMap;
 import net.ripe.db.whois.common.profiles.DeployedProfile;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Primary;
 import org.springframework.stereotype.Component;
 
-import javax.annotation.PostConstruct;
-import javax.annotation.PreDestroy;
 import java.net.InetAddress;
 import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
 
 @DeployedProfile
 @Primary
 @Component
 public class HazelcastPersonalObjectAccounting implements PersonalObjectAccounting {
     private static final Logger LOGGER = LoggerFactory.getLogger(HazelcastPersonalObjectAccounting.class);
 
-    private static IMap<InetAddress, Integer> counterMap;
+    private IMap<InetAddress, Integer> counterMap;
+    private HazelcastInstance hazelcastInstance;
 
-    private static volatile HazelcastInstance instance;
+    @Autowired
+    public HazelcastPersonalObjectAccounting(final HazelcastInstance hazelcastInstance) {
+        this.hazelcastInstance = hazelcastInstance;
+        this.counterMap =  hazelcastInstance.getMap("queriedPersonal");
 
-    static synchronized void startHazelcast() {
-        if (instance != null) {
-            throw new IllegalStateException("Hazelcast already started");
-        }
-
-        instance = Hazelcast.newHazelcastInstance(null);
-        counterMap = instance.getMap("queriedPersonal");
-    }
-
-    static void shutdownHazelcast() {
-        LOGGER.debug("Shutting down hazelcast instance");
-
-        instance.getLifecycleService().shutdown();
-        instance = null;
-    }
-
-    @PostConstruct
-    public void startService() {
-        startHazelcast();
-    }
-
-    @PreDestroy
-    public void stopService() {
-        shutdownHazelcast();
+        LOGGER.info("hazelcast instances : " + this.hazelcastInstance.getName() +  " members: " + this.hazelcastInstance.getCluster().getMembers());
```

Comment:
Use slf4j curly braces `{}` and a comma-separated list of values, don't use string concatenation